cmd/lakectl/cmd/local_init.go

```diff
@@ -0,0 +1,83 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"github.com/treeverse/lakefs/pkg/git"
+	"github.com/treeverse/lakefs/pkg/uri"
+)
+
+const (
+	initMinArgs = 1
+	initMaxArgs = 2
+)
+
+var localInitCmd = &cobra.Command{
+	Use:   "init [directory] <lakeFS path URI>",
```

Comment:
1. Optional usually comes at the end
2. lakeFS path URI - check other commands like `fs` we reference it as `path uri`